src/constants.js

```diff
@@ -121,6 +121,28 @@ export const styles = {
     position: "relative"
   },
 
+  panelPopupContainer: {
+    position: "absolute",
+    left: 20,
+    top: 20,
+    width: "calc(100% - 40px)",
+    height: "calc(100% - 40px)",
+    zIndex: 10
+  },
+
+  panelPopup: {
+    padding: 10,
+    backgroundColor: "white",
+    overflow: "hidden",
+    height: "100%",
+    boxSizing: "border-box",
+    display: "flex",
+    flexDirection: "column",
+    //opacity: 0.7,
+    border: "solid 1px black",
```

Comment:
How difficult would it be to style this like our master modal style (https://www.figma.com/file/hVEnbfL3Ii13Nzsp0xdGfV/Code-Design-System-1.0?node-id=0%3A1800)?

Reply:
Have we implemented this anywhere yet, so that I can borrow the styling?